packages/web-react/src/components/Dropdown/Dropdown.tsx

```diff
@@ -2,14 +2,14 @@ import React, { createElement, useRef, LegacyRef } from 'react';
 import classNames from 'classnames';
 import DropdownWrapper from './DropdownWrapper';
 import { useStyleProps } from '../../hooks';
-import { DropdownPlacements, SpiritDropdownProps } from '../../types';
+import { SpiritDropdownProps } from '../../types';
 import { useDropdown } from './useDropdown';
 import { useDropdownStyleProps } from './useDropdownStyleProps';
 import { useDropdownAriaProps } from './useDropdownAriaProps';
 
 const defaultProps = {
-  placement: DropdownPlacements.BOTTOM_LEFT,
   enableAutoClose: true,
+  placement: 'bottom-left',
```

Comment:
```suggestion
  placement: Placements.BOTTOM_LEFT,
```
Please, the constant should be used here.

Reply:
Fixed.